Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 ruby '2.0.0'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.3'
+gem 'rails', '~> 4.2.3'
```

Comment:
yeah we could set this to 4.2.4 to be more explicit. since we version-control our `Gemfile.lock`, adding the `~>` modifier will let us just do `bundle update rails` to get the latest future patch-level updates though.